src/components/toolbar/Toolbar.tsx

```diff
@@ -53,21 +50,17 @@ export interface ToolbarProps extends ToolbarBaseProps {
   popoverInteractionKind?: PopoverInteractionType;
 }
 
-export interface ToolbarItemProps extends ToolbarBaseProps {
-  id?: string;
+export interface ToolbarItemProps
+  extends ToolbarBaseProps,
+    Omit<ButtonProps, 'onClick' | 'tooltipProps'> {
```

Comment:
I would rather `Pick` some props that we know we need, rather than omit.

Each props that we pick should have a corresponding story.